_includes/footer.html

```diff
@@ -1,5 +1,6 @@
 <footer class="footer">
   <div class="site-width">
-    <img src="/images/flag-white-red.png" alt="Code for America Flag">
+    <div class="text-center"><a href="http://codeforamerica.org"><img src="/images/flag-white-red-trans.png" alt="Code for America Flag"></a></div>
+    <div class="text-center"><small>Copyright 2015 Code Island All Rights Reserved.</small></div>
```

Comment:
I'm not sure about © here we may need to use some (cc) terms. I'll look into it.